releases/release-1.23/release-notes/maps/pr-104482-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 104482
+releasenote:
+  text: Changed kubectl describe to compute Age of an event using the count and lastObservedTime
```

Comment:
age? Update to `count` and `lastObservedTime` OR `EventSeries.count` and `EventSeries.lastObservedTime`